### ISSUE TITLE
BERT: Loading SST dataset in bert.ipynb 

### ISSUE THREAD

**Issue by BerenLuthien** [dmlc/gluon-nlp#604]
Do we have a working demo with SST dataset ?
Changing https://gluon-nlp.mxnet.io/examples/sentence_embedding/bert.html so as to use SST-2 dataset.

`data_train = dataset.SSTDataset('dev', root='.')` gives `AttributeError: 'module' object has no attribute 'SSTDataset'`. I basically copy paste the 
```
@register(segment=['train', 'dev', 'test'])
class SSTDataset(GLUEDataset):
...
      return ['0', '1']
```
from
https://github.com/dmlc/gluon-nlp/blob/master/scripts/bert/dataset.py 

Many thanks.

**Comment by eric-haibin-lin** [comment]
Hi, `SSTDataset` is only available on the master branch. It is not included in the downloaded files if you get it from the website for the stable version. Could you visit the master version website https://gluon-nlp.mxnet.io/master/examples/sentence_embedding/bert.html and try it again?

**Comment by szha** [comment]
@BerenLuthien BERT on SST has been released as part of 0.6.0. Let me know if you need this issue reopened.
